sys/arm64/conf/GENERIC-MORELLO

```diff
@@ -0,0 +1,227 @@
+#
+# GENERIC -- Generic kernel configuration file for FreeBSD/arm64
```

Comment:
Can this include GENERIC and override the differing options?

Reply:
We need to remove quite a few drivers that don't build on Morello and will never be used, e.g. any SoC specific driver.

Reply:
I don't like duplicating all the options lines since we miss all changes to GENERIC. For example we are now missing my "enable TMPFS" change.

Can we at least factor out the options part of the config file into a std.foo and keep the devices list separate?